src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1587,6 +1587,8 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("DEBUG_FLOAT_ARRAY", 1.0f);
 		configure_stream_local("DEBUG_VECT", 1.0f);
 		configure_stream_local("DISTANCE_SENSOR", 0.5f);
+		configure_stream_local("ESC_INFO", 1.0f);
```

Comment:
Fair warning, we might end up reducing this rate in the normal mode.

Reply:
No problem.